third_party/Makefile

```diff
@@ -5,11 +5,16 @@
 
 SB_MICRO_PATH ?= "/usr/local"
 MPI_HOME_PATH ?= "/usr/local/mpi"
+HIP_HOME ?= "/opt/rocm/hip"
+RCCL_HOME ?= "/opt/rocm/rccl"
 
-.PHONY: all cutlass bandwidthTest nccl_tests perftest fio
+.PHONY: all rocm_all cutlass bandwidthTest nccl_tests perftest fio rccl_tests
 
-# Build all targets.
-all: cutlass bandwidthTest nccl_tests perftest fio
+# Build all targets for cuda.
+all: cutlass bandwidthTest nccl_tests perftest 
```

Comment:
I think we would have `make cuda`, `make rocm`, and `make all`.

`make all` should be `all: cuda rocm`

Reply:
ok, thanks for the comments